### PR TITLE
Update index.rst (Documentation)

### PR DESCRIPTION
**Pull request by vlilanl**
I updated the documentation for metaAssembly.

One potential issue is that the diagram only illustrates the short reads workflow, but I'm not sure how important that is.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,230 +1,232 @@
...
 ---------------------
 
-Third party software:  (This is included in the Docker image.)
+Third-party software: (This is included in the Docker image.)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- `metaSPades v3.15.0 <https://cab.spbu.ru/software/spades/>`_ (License: `GPLv2 <https://github.com/ablab/spades/blob/spades_3.15.0/assembler/GPLv2.txt>`_)
-- `BBTools:38.94 <https://jgi.doe.gov/data-and-tools/bbtools/>`_ (License: `BSD-3-Clause-LBNL <https://bitbucket.org/berkeleylab/jgi-bbtools/src/master/license.txt>`_)
+- `metaSPAdes v3.15.0 <https://cab.spbu.ru/software/spades/>`_ (License: `GPLv2 <https://github.com/ablab/spades/blob/spades_3.15.0/assembler/GPLv2.txt>`_)
+- `BBTools v38.94 <https://jgi.doe.gov/data-and-tools/bbtools/>`_ (License: `BSD-3-Clause-LBNL <https://bitbucket.org/berkeleylab/jgi-bbtools/src/master/license.txt>`_)
```
Comment (chienchi) [inline docs/index.rst:81]: v39.03
I think we should update line 33 and 43 of jgi_assembly.wdl  to v39.03 which JGI used. 
`bbtools_container = bbtools_container`

File: docs/index.rst [hunk 2]
```diff
@@ -1,230 +1,232 @@
...
 Workflow Dependencies
 ---------------------
 
-Third party software:  (This is included in the Docker image.)
+Third-party software: (This is included in the Docker image.)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- `metaSPades v3.15.0 <https://cab.spbu.ru/software/spades/>`_ (License: `GPLv2 <https://github.com/ablab/spades/blob/spades_3.15.0/assembler/GPLv2.txt>`_)
-- `BBTools:38.94 <https://jgi.doe.gov/data-and-tools/bbtools/>`_ (License: `BSD-3-Clause-LBNL <https://bitbucket.org/berkeleylab/jgi-bbtools/src/master/license.txt>`_)
+- `metaSPAdes v3.15.0 <https://cab.spbu.ru/software/spades/>`_ (License: `GPLv2 <https://github.com/ablab/spades/blob/spades_3.15.0/assembler/GPLv2.txt>`_)
```
Comment (chienchi) [inline docs/index.rst:80]: v4.0.0

File: docs/index.rst [hunk 3]
```diff
@@ -1,230 +1,232 @@
...
-The workflow from GitHub uses all the listed docker images to run all third-party tools.
-The workflow is available in GitHub: https://github.com/microbiomedata/metaAssembly; the corresponding Docker images are available in DockerHub: https://hub.docker.com/r/microbiomedata/spades and https://hub.docker.com/r/microbiomedata/bbtools
+The workflow from GitHub uses all the listed Docker images to run all third-party tools.  
+
+The workflow is available on GitHub: `https://github.com/microbiomedata/metaAssembly`  
+
+The corresponding Docker images are available on DockerHub:
+
+- `https://hub.docker.com/r/microbiomedata/spades`
+- `https://hub.docker.com/r/microbiomedata/bbtools`
```
Comment (chienchi) [inline docs/index.rst:45]: These are for long reads and should be added.
https://hub.docker.com/r/staphb/flye
https://hub.docker.com/r/staphb/racon
https://hub.docker.com/r/staphb/minimap2
https://hub.docker.com/r/staphb/samtools

**Review by chienchi — CHANGES_REQUESTED** [review]
Thank you. Looks good. A few updates.

**Comment by chienchi** [comment]
Also, please help updating the README.md  which could be a brief version of the index.rst and point to [doc site ](https://nmdc-workflow-documentation.readthedocs.io/en/latest/chapters/3_MetaGAssemly_index.html) for more detailed info.

**Review by chienchi — APPROVED** [review]

**Comment by vlilanl** [comment]
Jaws test for updated bbtools container is 91505

**Review by chienchi — APPROVED** [review]

**Comment by aclum** [comment]
@vlilanl sorry for the late review. Would it be possible to show that there are two rounds of polishing with Racon in the image overview?

**Pull request merged**